CHANGELOG.md

```diff
@@ -8,6 +8,13 @@ Semantic versioning in our case means:
 - Minor releases do bring new features and configuration options. New violations can be added. Code that passes `x.0.y` might not pass on `x.1.y` release.
 - Major releases inidicate significant milestones or serious breaking changes.
 
+## 0.16.0
+
+### Features
+
```

Comment:
Can you please also add to `Bugfixes` part of the changelog that we had fixed an issue with unpacking lists?

Reply:
Well, right now `WrongUnpackingViolation` is as violation for tuples only. And with this changes, it will will be raised for lists too. Maybe this is more a feature than a bugfix?

Reply:
Both are fine!